web/src/screens/private/Schedules/EditSchedule.js

```diff
@@ -130,6 +113,9 @@ class AddSchedule extends React.Component {
       .filter(tag => tag.type === TAG_TYPE_CAPABILITY)
       .map(tag => ({ value: tag.id.toString(), label: tag.name }));
 
+    const groups = user.groups
+      .map(group => ({ value: group.id.toString(), label: group.name }));
```

Comment:
toString() will shit a brick if id is every null compared to String() which should not. id should never be null thanks to props validation but something to keep in mind

```
> let value = null
undefined
> value.toString()
TypeError: Cannot read property 'toString' of null
> String(value)
'null'
```